### PR TITLE
Ref

### PR DESCRIPTION
**Pull request by Abhijithchintu**
Integrated mongo db

### CODE REVIEW

**Review by manu156 — DISMISSED** [review]

File: Chat/.env [hunk 1]
```diff
@@ -1 +1,2 @@
-PORT="3002"
\ No newline at end of file
+PORT="3002"
+MONGO_URI="mongodb+srv://abhichintu7:Mongo%401234@cluster0.g0z1y7n.mongodb.net/?retryWrites=true&w=majority"
```
Comment (manu156) [inline Chat/.env:5]: can we deploy this on local?

File: Chat/db/mongodb/schema.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+const messageSchema = new mongoose.Schema({
+    client_id: Number,
+    user_name: String,
```
Comment (manu156) [inline Chat/db/mongodb/schema.js]: what is the use of user_name?

File: Chat/db/mongodb/schema.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    sender_name: String,
+    receiver_name: String,
```
Comment (manu156) [inline Chat/db/mongodb/schema.js]: can we use receiver_client_id instead of name?

**Review by manu156 — CHANGES_REQUESTED** [review]

**Review by AnuguRakeshReddy — DISMISSED** [review]
Do better coding, idiot!

**Review by manu156 — DISMISSED** [review]

**Review by manu156 — APPROVED** [review]

**Comment by manu156** [comment]
resolved merge conflicts @Abhijithchintu . merging.

**Pull request merged**